contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -389,13 +383,11 @@ abstract contract PoSValidatorManager is
         }
 
         address owner = $._posValidatorInfo[validationID].owner;
-
         address rewardRecipient = $._rewardRecipients[validationID];
 
+        // the reward-recipient should always be set, but just in case it isn't, we won't burn the reward
         if (rewardRecipient == address(0)) {
             rewardRecipient = owner;
-        } else {
-            delete $._rewardRecipients[validationID];
```

Comment:
I think @iansuvak's [comment](https://github.com/ava-labs/teleporter/pull/657/files#r1854548830) is meant to go here.